README.md

```diff
@@ -63,7 +63,9 @@ For more information about spotless checks see
     s3:
       endpointUrl: http://minio:9000
       region: local
-      bucket: ingest-quarantine
+      bucket:
```

Comment:
Someday we may decide that one kind of bucket is fine for any type of Data of we want to persist to S3. Someday.